File: WebServer_0001/WindowsFormsWebServer/Model/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WindowsFormsWebServer.Model
{
    public class Post
    {
        public int Id { get; set; }
        public int userId { get; set; }
        public string title { get; set; }
        public string body { get; set; }

        public Post()
        {

        }

        public Post( int userId, string title, string body)
        {
            this.userId = userId;
            this.title = title;
            this.body = body;
        }
    }
}

File: WebServer_0001/WindowsAppWebServer/Model/Post.vb
Public Class Post
    Public Property Id As Integer
    Public Property userId As Integer
    Public Property title As String
    Public Property body As String

    Public Sub New()
    End Sub

    Public Sub New(ByVal userId As Integer, ByVal title As String, ByVal body As String)
        Me.userId = userId
        Me.title = title
        Me.body = body
    End Sub
End Class
